packages/web-forms/src/components/common/CheckboxWidget.vue

```diff
@@ -13,6 +13,9 @@ defineEmits(['update:modelValue', 'change']);
 const props = defineProps<CheckboxWidgetProps>();
 
 const selectValues = (values: readonly string[]) => {
+	if (props.question.appearances.label) {
```

Comment:
Feels like it'd be great to share that logic between select one and multiple. But that can be for another time!

Reply:
This morning, I am considering that when it's `labels` appearance, it should render simple text and not use radio or checkboxes (and we delete this condition). I will open a ticket for that improvement; it will require a bit CSS work.